tests/RangeTest.java

```diff
@@ -0,0 +1,157 @@
+import org.junit.*;
+import util.*;
+
+public class RangeTest {
```

Comment:
Each comment you put inside a unit test seems to indicate the start of a new unit test and so you should break them to smaller unit tests with one meaning each.